platform-service/src/unstract/platform_service/controller/platform.py

```diff
@@ -20,12 +20,19 @@
 
 from unstract.flags.feature_flag import check_feature_flag_status
 
-be_db = peewee.PostgresqlDatabase(
+be_db = PooledPostgresqlDatabase(
```

Comment:
Do we need to rely on pgbouncer for these services if we perform connection pooling?
cc: @harini-venkataraman @hari-kuriakose @ritwik-g

Reply:
Ideally we do not. But for the time being it's okay.

Reply:
@chandrasekharan-zipstack We do not rely on pg bouncer. This handles application level pooling.